src/native/corehost/fxr/command_line.cpp

```diff
@@ -158,10 +158,10 @@ namespace
 
             app_candidate = argv[*new_argoff];
 
-            bool is_app_managed = utils::ends_with(app_candidate, _X(".dll"), false) || utils::ends_with(app_candidate, _X(".exe"), false);
+            bool is_app_managed = utils::ends_with(app_candidate, _X(".dll"), false);
```

Comment:
Is it currently possible for someone to pass a .NET Framework EXE and have it run?